packaging/ubuntu-16.04/rules

```diff
@@ -270,3 +270,17 @@ override_dh_auto_clean:
 
 override_dh_gencontrol:
 	dh_gencontrol -- -VBuilt-Using="$(BUILT_USING)" $(SUBSTVARS)
+
+override_dh_systemd_start:
+	# same order as automatically discovered units, but make sure snap.mount is
+	# handled externally
```

Comment:
Is there a sanity check we can add to ensure that the list of automatically generated units is what it was at the time when this was coded so when a new unit name is added this part will fail and someone will notice and make a decision?